tests/unit/s2n_extension_type_test.c

```diff
@@ -517,7 +517,12 @@ int main()
 
                 /* Expected CLIENT_HELLO extensions sent, but not received */
                 EXPECT_OK(s2n_negotiate_test_server_and_client_until_message(server_conn, client_conn, SERVER_HELLO));
-                EXPECT_TRUE(S2N_CBIT_TEST(client_conn->extension_requests_sent, key_shares_id));
+                if (s2n_is_tls13_fully_supported()) {
+                    EXPECT_TRUE(S2N_CBIT_TEST(client_conn->extension_requests_sent, key_shares_id));
+                } else {
+                    EXPECT_FALSE(S2N_CBIT_TEST(client_conn->extension_requests_sent, key_shares_id));
+                }
+
```

Comment:
This  makes the test harder to follow, and we test TLS1.2 else where. Let's just restrict it to TLS1.3.

Reply:
Fixed